README.md

```diff
@@ -59,7 +59,7 @@ edf = EDF.File(my_edf_file_path)
 plans = plan_edf_to_onda_samples(edf; label=my_labels)
 
 function fix_millivolts(plan)
-    if plan.sample_unit == "millivolt" && plan.kind == "eeg"
+    if plan.sample_unit == "millivolt" && plan.sensor_type == "eeg"
```

Comment:
Were these untested? How did we do the initial upgrade to Legolas 0.5 without this failing?

Reply:
First, these are in OndaEDF, not OndaEDFSchemas (which is where the initial upgrade was confined to).

Second, these are just examples in the README, so no, they're not tested.

Reply:
READMEs